ts/Extensions/Annotations/NavigationBindings.ts

```diff
@@ -228,6 +235,46 @@ const bindingsUtils = {
         } as Record<string, ('checkbox'|'number'|'text')>)[
             typeof value
         ];
+    },
+    /**
+     * Returns the first xAxis or yAxis that was clicked with its value.
+     *
+     * @private
+     * @function Highcharts.NavigationBindingsUtilsObject#getAssignedAxis
+     *
+     * @param {Highcharts.PointerAxisCoordinatesObject} coords
+     *        All the chart's axes with a current pointer's axis value.
+     *
+     * @param {string} xOrY
+     *        'x' or 'y' indicating axis.
+     *
+     * @return {Highcharts.PointerAxisCoordinateObject}
+     *         Object with a first found axis and its value that pointer
+     *         is currently pointing.
+     */
+    getAssignedAxis(
+        coords: Highcharts.PointerAxisCoordinatesObject,
+        xOrY: ('x'|'y')
```

Comment:
I find the `xOrY` parameter name a bit odd. Couldn't it be something like `isXAxis: boolean` like in Axis class?

Reply:
`axis.isXAxis` would not be useful in this case, because I am not operating on a specific axis, but on an object like this:
```js
coords = {
    xAxis: [{axis: axis, value: number}, ...],
    yAxis: [{axis: axis, value: number}, ...]
}
```

However, when I'm looking at this code after few days, I see that I don't need to loop through both properties `xAxis` and `yAxis` in `objectEach` there, because I am returning only one of them and I know which one from the beginning. I'll take a closer look at it tomorrow and probably improve.